coco/worker.py

```diff
@@ -165,8 +165,25 @@ async def go():
 
             # Always attempt to return the result so that the client doesn't hang...
             finally:
-                await conn.execute("rpush", f"{name}:res", json.dumps(result))
-                await conn.execute("rpush", f"{name}:code", code)
+                while True:
```

Comment:
I'm not sure about putting this in an infinite loop, seems like we're just adding an opportunity for it to get stuck without logging anything. I think we both agree that it shouldn't give you this error more than once, but if that's true why put in a loop.

Reply:
good point